lib/rules/web/admin_console/4.1/check_links.xyaml

```diff
@@ -2,4 +2,16 @@ check_doc_get_started_with_cli:
   elements:
   - selector:
       xpath: //.[contains(@href, "/k8s/cluster/projects")]/button[contains(., "View My Projects")]
-
+check_header_brand:
+  elements:
+  - selector:
+      xpath: //a/img[@alt='<product_brand>']
+click_help_on_masthead:
+  elements:
+  - selector:
+      xpath: //button[@aria-label="Help"]
+    op: click
+check_doc_link_on_page:
+  elements:
+  - selector:
+      xpath: //a[contains(@href, '<doc_url>')]
```

Comment:
It looks like matching a normal `//a` with given link `'<doc_url>'`, can we add strict check?